pkg/metadata/host/host.go

```diff
@@ -55,7 +56,7 @@ func GetPayload(hostname string) *Payload {
 		SystemStats:   getSystemStats(),
 		Meta:          meta,
 		HostTags:      getHostTags(),
-		ContainerMeta: getContainerMeta(),
+		ContainerMeta: getContainerMeta(time.Second),
```

Comment:
same here, could we user `1*time,Second` ? Also do we want to make that timeout configurable ?